cmd/config/repo.go

```diff
@@ -0,0 +1,29 @@
+package config
+
+import (
+	"context"
+	"strings"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/Shopify/voucher/repository"
+	"github.com/Shopify/voucher/repository/github"
+)
+
+func newRepositoryClient(org repository.Organization) (repositoryClient repository.Client) {
+	if strings.Contains(org.URL, "github.com") {
+		keyring, err := getRepositoryKeyRing()
+		if nil != err {
+			log.Errorf("failed to get github token: %s", err)
+			return nil
+		}
+		token := keyring["github"]
```

Comment:
Could we change the token to match the domain of the server?
```suggestion
		token := keyring["github.com/Shopify"]
```

Then we can select the token based on the whole domain?

```golang
for domain, token := range keyring {
    if strings.Contains(org.URL, domain) {
         if isGithubDomain(org.URL) {
              return gitHub.NewClient(ctx, &token)
         }

    } 
}
```

Reply:
The statement [before](https://github.com/Shopify/voucher/pull/62/files/65d50f709c403e33f30fac1059b99229ebb015db#diff-d121569490d02cdb701b44da27084ac9R14): `if strings.Contains(org.URL, "github.com") {` checks that it is a Github domain before getting the keyring. Should it be rechecked by `isGithubDomain`? Or would it require some other checks?

Reply:
We might have situations where we have multiple GitHub organizations, which could require different GitHub tokens, especially when it comes to orgs that have multiple GitHub environ
ments (eg. the Grafeas org in Github vs the Google org in Github).

Reply:
But what would `isGithubDomain(org.URL)` check do that this check `if strings.Contains(org.URL, "github.com")` doesn't? 
It doesn't seem like `org.URL` depends on loop variables, so why is this check inside?

Reply:
> But what would isGithubDomain(org.URL) check do that this check if strings.Contains(org.URL, "github.com")

I think it's totally reasonable to just do that!

> It doesn't seem like org.URL depends on loop variables, so why is this check inside?

it doesn't necessarily need to be! But it might be easier to iterate over the available tokens rather than iterate over the different organizations, you know?

Reply:
Done.